README.md

```diff
@@ -1,17 +1,15 @@
 # Fork Awesome
 ### A fork of the iconic font and CSS toolkit
 
-[![npm-badge]][npm-link] [![all-contrib]](CONTRIBUTORS.md) [![jsdeliver-badge]][jsdeliver-link] [![cdnjs-badge]][cdnjs-link] [![build-status-badge]][build-status-link]
+[![npm-badge]][npm-link] [![all-contrib]](CONTRIBUTORS.md) [![jsdeliver-badge]][jsdeliver-link] [![cdnjs-badge]][cdnjs-link] [![build-status-badge]][build-status-link] [![chat-badge]][chat-link]
 
-Fork Awesome is a full suite of 796 pictographic icons for easy scalable vector graphics on websites, originally created by [Dave Gandy](https://twitter.com/davegandy) and now maintained by a community.
+**Fork Awesome is a suite of 796 pictographic and brand icons for easy, scalable vector graphics on websites and beyond.**
 
-Following concerns regarding [the development of Font Awesome](https://github.com/FortAwesome/Font-Awesome/issues/12199#issuecomment-362919956), the PR Freeze since Oct 2016 and the direction [Fort Awesome](https://fortawesome.com/) is taking with the version 5.0 of their project, we are forking Font Awesome (4.7), in order to build on this amazing tool Dave Gandy has given us, while at the same time allowing this project to be run by a distributed community of contributors.
+This project, as the name suggests, is a fork of [Font Awesome](https://fontawesome.com). Font Awesome was originally created by [Dave Gandy](https://twitter.com/davegandy) as a community project. However, as Font Awesome developed, PRs from the community stopped being accepted (Oct. 2016) and as of version 5.0 [the build system became private](https://github.com/FortAwesome/Font-Awesome/issues/12199#issuecomment-362919956) (Feb. 2018).
 
-A tremendous gratitude is given to the whole team behind [Font Awesome](https://fontawesome.com), and you are encouraged to support them and buy Font Awesome Pro for your project.
+With gratitude to Dave Gandy and the Font Awesome team, [Julien](https://github.com/xuv) [Deswaef](@xuv@merveilles.town) forked Font Awesome 4.7 into Fork Awesome 1.0 in [Feb. 2018](https://github.com/ForkAwesome/Fork-Awesome/releases/tag/1.0.0), to continue building on the amazing resource in a fully reproduceable way, as a distributed community of contributors.
```

Comment:
@xuv@merveilles.town is not a valid URL. Perhaps you meant

```suggestion
With gratitude to Dave Gandy and the Font Awesome team, [Julien](https://github.com/xuv) [Deswaef](https://merveilles.town/@xuv) forked Font Awesome 4.7 into Fork Awesome 1.0 in [Feb. 2018](https://github.com/ForkAwesome/Fork-Awesome/releases/tag/1.0.0), to continue building on the amazing resource in a fully reproduceable way, as a distributed community of contributors.
```

Some more updates to the content

```suggestion
With gratitude to Dave Gandy and the Font Awesome team, [Julien](https://github.com/xuv) [Deswaef](@xuv@merveilles.town) forked Font Awesome 4.7 into [Fork Awesome 1.0](https://github.com/ForkAwesome/Fork-Awesome/releases/tag/1.0.0) in February 2018, to continue building on the amazing resource in a truly open source fashion; with the distributed community of contributors and for the community.
```